compiler/src/dotty/tools/dotc/typer/Implicits.scala

```diff
@@ -1371,8 +1376,12 @@ trait Implicits:
         history match
           case prev @ OpenSearch(cand1, tp, outer) =>
             if cand1.ref eq cand.ref then
+              def checkFullyDefined =
+                val result = isFullyDefined(tp, ForceDegree.failBottom)
```

Comment:
What would happen if we forced the type to be fully defined at that point instead of disqualifying it?

Reply:
I guess it will only be disqualified if a type variable would be instantied to Nothing, but that might be a legitimate instantiation

Reply:
Yes, I don't know what is the right force degree, or whether there even is one.

Reply:
I've tried setting the force degree to all which changes how we do error reporting but seems to fine, let me know what you think.

Reply:
I think that's fine, too. It's a corner case, so the simpler the handling the better.